src/types/display_data.rs

```diff
@@ -27,6 +30,55 @@ bitflags! {
     }
```

Comment:
I dont think it's a good idea to have bitflags when each flag marks a single bit AND all bits can be used to make any byte.
I think it's rather pointless, but it could stay for backwards compatibility.